homeassistant/components/jewish_calendar/quality_scale.yaml

```diff
@@ -0,0 +1,92 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup:
+    status: exempt
+    comment: Entities of this integration do not explicitly subscribe to events.
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure:
+    status: exempt
+    comment: Local calculation does not require configuration.
+  test-before-setup:
+    status: exempt
+    comment: Local calculation does not require setup.
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions: done
```

Comment:
```py
            _LOGGER.error("Can't get sunset event date for %s", today)
            raise HomeAssistantError(
                translation_domain=DOMAIN, translation_key="sunset_event"
            )
```
why do we both log and raise? Can we enrich the error message so we don't need to do both?

Reply:
Originally I had only the error there, I added the raise due to the actions-exceptions.
Which is preferrable?

Reply:
Done